tests/test_load_video_frames.py

```diff
@@ -458,7 +458,9 @@ def test_megadetector_lite_yolox_dog(tmp_path):
             "-y",
         ]
     )
-    frames = load_video_frames(tmp_path / "dog.mp4", megadetector_lite_config={"confidence": 0.25})
+    frames = load_video_frames(
+        tmp_path / "dog.mp4", megadetector_lite_config=MegadetectorLiteYoloXConfig()
```

Comment:
the default is a confidence of 0.25 so this isn't changing the underlying behavior